src/premiscale/hypervisor/qemu_data.py

```diff
@@ -284,17 +284,100 @@ def to_tinyflux(self) -> Tuple[Dict, Dict, Dict, Dict]:
             _block_datum
         )
 
-    def to_influx(self) -> Dict:
+    def to_influx(self) -> Tuple[Dict, Dict, Dict, Dict]:
```

Comment:
Another problem is that this isn't being automatically called based on the backend in-use.